Makefile

```diff
@@ -168,18 +175,18 @@ dist_unportable_release:
 # submitting pull requests. Note though that the CI system tests TiKV
 # through its own scripts and does not use this rule.
 test:
-        # When SIP is enabled, DYLD_LIBRARY_PATH will not work in subshell, so we have to set it
-        # again here. LOCAL_DIR is defined in .travis.yml.
-        # The special linux case below is testing the mem-profiling
-        # features in tikv_alloc, which are marked #[ignore] since
-        # they require special compile-time and run-time setup
-        # Forturately rebuilding with the mem-profiling feature will only
-        # rebuild starting at jemalloc-sys.
+	# When SIP is enabled, DYLD_LIBRARY_PATH will not work in subshell, so we have to set it
+	# again here. LOCAL_DIR is defined in .travis.yml.
+	# The special linux case below is testing the mem-profiling
+	# features in tikv_alloc, which are marked #[ignore] since
+	# they require special compile-time and run-time setup
+	# Forturately rebuilding with the mem-profiling feature will only
+	# rebuild starting at jemalloc-sys.
 	export DYLD_LIBRARY_PATH="${DYLD_LIBRARY_PATH}:${LOCAL_DIR}/lib" && \
 	export LOG_LEVEL=DEBUG && \
 	export RUST_BACKTRACE=1 && \
 	cargo test --no-default-features --features "${ENABLE_FEATURES}" --all ${EXTRA_CARGO_ARGS} -- --nocapture $(TEST_THREADS) && \
-	cargo test --no-default-features --features "${ENABLE_FEATURES}" --all --bench misc ${EXTRA_CARGO_ARGS} -- --nocapture  $(TEST_THREADS) && \
+	cargo test --no-default-features --features "${ENABLE_FEATURES}" -p tests --bench misc ${EXTRA_CARGO_ARGS} -- --nocapture  $(TEST_THREADS) && \
```

Comment:
So benchmarks inside components will be ignored?

Reply:
No, this command only tests, it does not bench, and previously it only built and tested the `misc` bench, which is what it still does.